packages/extension-ui/src/Popup/About.tsx

```diff
@@ -22,7 +24,7 @@ const About = () => {
           headerText={t<string>('Aleph Zero Signer')}
           iconType='aleph'
         >
-          {t<string>('Version')}&nbsp;{t<string>('version-number')}
+          {t<string>('Version')}&nbsp;{SIGNER_VERSION}
```

Comment:
doesn't webpack expose values from package.json? Would that work with replace - I'm not sure

Reply:
That might actually work after I removed all the generated files 👍